pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,77 @@
+{ stdenv, fetchFromGitHub, fetchurl, lib
+, mariadb-connector-c, boost, cmake, bison, flex
+}:
+
+let
+  columnar = fetchFromGitHub {
+    owner = "manticoresoftware";
+    repo = "columnar";
+    rev = "b211815f";
+    sha256 = "sha256-7oWqa1P1sGUkaTuMLWz36KhR42SjEWQo/4g+JKAsJOU=";
+  };
+  galera_branch = "5e7af6e5";
```

Comment:
We cannot rely on short hashes.

Also for fetching from github please use fetchFromGitHub

Reply:
I'm using fetchurl so that I can fetch a zip of the galera sourcecode and place it in the location that their cmake files expect to find it (https://github.com/manticoresoftware/manticoresearch/blob/5.0.2/cmake/GetGALERA.cmake#L9).

Would it be better to use fetchFromGitHub to fetch a directory of the sourcecode, then `substituteInPlace` to set [GALERA_PLACE](https://github.com/manticoresoftware/manticoresearch/blob/348514c866092de6bc7d3c201a9960e0e462c3ba/cmake/GetGALERA.cmake#L46) from that?

Reply:
I've got an alternate version that uses mkDerivation to build all the subprojects rather than having manticore's build process unpack & build them itself - would that be preferred? https://github.com/jdelStrother/nixpkgs/commit/29f27f742b09cf4e54b75184ccba86fe83e2392b

Reply:
> jdelStrother@29f27f7

that looks good except it should use pname+version and some basic meta.